frontend/src/util/activity.js

```diff
@@ -16,7 +16,11 @@ const createActivityEvent = (message) => {
 }
 
 const createSingleMessageActivity = (activityParams, messageList) => {
-  if (messageList.length !== 1) throw new Error('Must have at exactly one message')
+  if (messageList.length !== 1) {
+    console.error('Expected the message list to be of size 1 in createSingleMessageActivity')
+    return null
+  }
+
```

Comment:
I do not think returning null here is a good idea as we only postpone the problem.
As those tests were "assert-like" tests (meaning this won't happen now that the code is tested) I would rather totally remove them.